projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -41,18 +50,29 @@ class JobStatus(str, Enum):
 
 
 class VDKHook(HttpHook):
+    """
+    Interact with Verstile Data Kit (VDK) Control Service
+    """
+
+    conn_name_attr = "vdk_conn_id"
+    default_conn_name = "vdk_default"
+    conn_type = "vdk"
```

Comment:
isn't it 'http'?

Reply:
The HttpHook has `conn_type="http"` https://github.com/apache/airflow/blob/047a6162b0b4cbf07fe2fd978e335839a7d3900b/airflow/providers/http/hooks/http.py#L41, so it didn't make sense to me to assign the same connection type.